### DISCUSSION TITLE
Can make it with any template ?

### DISCUSSION THREAD

**Discussion by prawee** [kool-dev/kool#378] [Ideas] [answered]
i want to make it with my template  
such as

```bash
kool create nextjs  my-project --template=initial-project
```

**Comment by fabriciojs** [comment, marked as answer]
Hello @prawee !

Right now we don't have this ability - to pass in arbitrary extra options to the `kool create` command. It would certainly be interesting to have, I believe, as many "create project" CLI do have specific/edge options that one may want to use.

So we are talking to have the option to pass in any extra arguments to `kool create` and then proxy them to the commands used for the preset `create:` config.

What do you guys think? cc @danielsuguimoto @dbpolito @fbrissi  (thinking about Java presets)

**Reply by prawee** [reply]
okay thank you.

**Reply by danielsuguimoto** [reply]
parsing extra args on `kool create` sounds good for me...
